packages/twenty-server/src/engine/metadata-modules/agent/constants/agent-system-prompts.const.ts

```diff
@@ -68,4 +66,18 @@ Guidelines:
 - Preserve the context and meaning from the original execution results
 - Ensure the output is clean, well-formatted, and ready for workflow consumption
 - Pay special attention to any data returned from tool executions (database queries, record creation, etc.)`,
+
```

Comment:
As discussed it's okay to have a system prompt but the node prompt should also be taken into account